crs/v1alpha2/controlplane_ubuntu.yaml

```diff
@@ -96,3 +103,15 @@ spec:
         owner: root:root
         permissions: '0644'
         content: ${SSH_PUB_KEY_CONTENT}
+      - path : /etc/netplan/60-ironicendpoint.yaml
+        owner: root:root
+        permissions: '0644'
+        content: |
+          network:
+            version: 2
+            renderer: networkd
+            bridges:
+              ironicendpoint:
+                interfaces: [enp1s0]
+                dhcp4: yes
```

Comment:
Aren't you going to have two IP addresses on that network since you have dhcp enabled on enp1s0 in previous netplan ?

Reply:
True ... I have removed it from the previous netplan